src/openepd/model/specs/generated/enums.py

```diff
@@ -208,6 +208,7 @@ class GypsumFireRating(StrEnum):
     X = "X"
     C = "C"
     F = "F"
+    OTHER = "Other"
```

Comment:
Provide a docstring in a correct place, so it will be propagated to an API docs.

Reply:
Done